go/tools/release/github/releases.go

```diff
@@ -59,6 +60,14 @@ func Releases(user, repo, token string) ([]Release, error) {
 	return releases, nil
 }
 
+func LatestRelease(user, repo, token string) (*Release, error) {
+	if token != "" {
+		token = "?access_token=" + token
```

Comment:
Use the [Go API](https://golang.org/pkg/net/url/#URL) to piece together this URL.